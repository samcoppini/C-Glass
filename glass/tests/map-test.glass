{M
    [m
        (_s)(String)!
        (_o)(Output)!

        (_s)(cmp).(_strMap)(Map)!

        "foo""bar"(_strMap)(set).?
        "foooo""!!!!"(_strMap)(set).?

        "Testing Map.has:"(_o)(ol).?
        "bar"(_strMap)(has).?(_o)(onl).?
        "foo"(_strMap)(has).?(_o)(onl).?

        "Testing Map.get:"(_o)(ol).?
        "foo"(_strMap)(get).?(_o)(ol).?
        "foooo"(_strMap)(get).?(_o)(ol).?
        "foo""quux"(_strMap)(set).?
        "foo"(_strMap)(get).?(_o)(ol).?
    ]
}
