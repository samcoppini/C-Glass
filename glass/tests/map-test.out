Testing Map.has:
0
1
Testing Map.get:
bar
!!!!
quux
